lib/Test/Class.pm

```diff
@@ -298,7 +298,11 @@ sub _run_method {
                 if $exception;
     } elsif ($num_done > $num_expected) {
         my $class = ref $self;
-        $Builder->diag("expected $num_expected test(s) in $class\::$method, $num_done completed\n");
+        if ($self->fail_if_returned_late) {
+            $Builder->ok(0, "($class\::$method returned before plan complete)");
```

Comment:
This message is not correct because method returned later then plan was completed.

I suggest to use similar/same message as is for diag output:
```perl
$Builder->ok(0, "expected $num_expected test(s) in $class\::$method, $num_done completed");
```

Reply:
Thanks for the suggestion! This has been corrected.